apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx

```diff
@@ -401,14 +366,6 @@ const SwapForm = () => {
     );
   }
 
-  // TODO: ensure that the error is catch by Sentry in this case
-  if (storedProviders?.length === 0 || providersError) {
```

Comment:
I am not sure to understand this logic know...if providers.length === 0. Are we going to show a loading state?
What happens if backend is failing and we do not have providers?

Reply:
So I believe that this is now managed by the usePageState. So if calls were to fail, or go to a loading state, then that hook would manage it.